deployments/container/Dockerfile.ubi8

```diff
@@ -14,23 +14,25 @@
 
 ARG GOLANG_VERSION=1.20.5
 ARG CUDA_IMAGE=cuda
-ARG CUDA_VERSION=12.2.0
+ARG CUDA_VERSION=12.3.0
 ARG BASE_DIST=ubi8
 FROM nvidia/cuda:${CUDA_VERSION}-base-${BASE_DIST} as build
 
+ARG GOLANG_VERSION=1.20.5
```

Comment:
Why the duplicate version?

Reply:
ah ups, I removed it on ubuntu, and forgot to remove it here. the ARG GOLANG_VERSION needs to be defined after FROM, or the docker build fail.